osbs/api.py

```diff
@@ -515,6 +515,7 @@ def _do_create_prod_build(self, git_uri, git_ref,
             platform_node_selector=self.build_conf.get_platform_node_selector(platform),
             filesystem_koji_task_id=filesystem_koji_task_id,
             koji_upload_dir=koji_upload_dir,
+            platform_descriptors=self.build_conf.get_platform_descriptors,
```

Comment:
`self.build_conf.get_platform_descriptors` is a reference to the method, did you mean `self.build_conf.get_platform_descriptors()`